### DISCUSSION TITLE
You can now install apps from WAU with Intune/SCCM

### DISCUSSION THREAD

**Discussion by Romanitho** [Romanitho/Winget-AutoUpdate#576] [Announcements]
To install apps with Intune/SCCM, you could use this repo:
https://github.com/Romanitho/Winget-Install
But... let's imagine we need to deploy an update of Winget-Install for all of our packages... time consuming...
Now, Winget-Install is part of WAU... Just call this as your install command:
`%programdata%\Winget-AutoUpdate\Winget-Install.ps1 -AppIDs 7zip.7zip`
(For intune, juste create an empty .intunewim)
Put this package WAU as dependence, and that's it.
Let's WAU self update or deploy new WAU updates as you want...

Thanks @KnifMelti for the orignal idea ;)

**Comment by 4f1sh3r** [comment]
I would love to test this out, but unfortunately I am always getting "Failed to install" (Status Code 0x800700C1) and the install.log of WAU doesn't even have new entries, so I guess Intune is unable to execute the script. I can confirm that the path is correct and it works when I execute it manually. The InTuneManagementExtension.log doesn't show anything useful. Any other ideas?

I am using the latest nightly of WAU, 1.19.2-10

**Reply by 4f1sh3r** [reply]
Got it working by using `"%systemroot%\sysnative\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -ExecutionPolicy Bypass -File %programdata%\Winget-AutoUpdate\Winget-Install.ps1 -AppIDs 7zip.7zip `
Works perfectly fine now, thanks for the great project!

**Reply by Romanitho** [reply]
Thanks for your feedback. Indeed we need to call PowerShell 64 with Intune... I forgot to mention this.
